run_dir/static/css/status.css

```diff
@@ -1075,3 +1075,15 @@ div.setOverflow {
 .close{
   margin-top:-10px;
 }
+
+.source li {
+  margin: 5px;
+  padding: 5px;
+  -webkit-border-radius: 4px;
+  -moz-border-radius: 4px;
+  border-radius: 4px;
+  text-shadow: 0 1px 0 rgba(255, 255, 255, 0.5);
+  background-color: #ddd;
+  border: 1px solid #ddd;
+  color: black;
+}
```

Comment:
Can we also add some change of cursor when hoovering here?

Reply:
Done! 👍